tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -520,22 +526,20 @@ TEST(TfGeometry, Quaternion)
     q1.header.frame_id = "A";
 
     // simple api
-    const geometry_msgs::msg::QuaternionStamped q_simple = tf_buffer->transform(
-      q1, "B", tf2::durationFromSec(
-        2.0));
-    EXPECT_NEAR(q_simple.quaternion.x, M_SQRT1_2, EPS);
-    EXPECT_NEAR(q_simple.quaternion.y, 0, EPS);
-    EXPECT_NEAR(q_simple.quaternion.z, -1 * M_SQRT1_2, EPS);
-    EXPECT_NEAR(q_simple.quaternion.w, 0, EPS);
+    {
```

Comment:
What are the scope brackets for?

Reply:
Protection against accidentally using `q_simple` in the `q_advanced` block - a bit of an idiosyncrasy of mine, but it helps with not making copy/paste errors.

Reply:
uhg, gtest is trash. I really prefer each test being a discrete function. I'll allow it.